chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/search/CertifiedProductFlatSearchResult.java

```diff
@@ -5,6 +5,11 @@
 public class CertifiedProductFlatSearchResult extends CertifiedProductSearchResult {
     private static final long serialVersionUID = -2547390525592841044L;
 
+    @JsonView({
+        SearchViews.Default.class
+    })
+    private String developerStatus;
```

Comment:
Why is there anything in this class? I put it here because the ticket wanted the status in the search results page only, so this seemed appropriate. I could move it to the super class, but if we did that why wouldn't the other things in this class also be in the super class?

Reply:
This class is for those fields we "flatten" with an emoji delimiter for the website, like the list of cqms. In the search results that come back via the API, like if you wanted to do a search with Postman, those fields come back as a JSON array more like you'd expect.

Reply:
Oh, that makes sense